assets/prototype/domain/shared/predicates/tickets/filters/validFiniteQuantity.ts

```diff
@@ -1,9 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import { Ticket } from '../../../../eventEditor/data/types';
+
 /**
  * @param {Object} ticket    event ticket object
  * @return {boolean} true if qty property is valid and NOT infinite
  */
-const validFiniteQuantity = (ticket) => {
-	return ticket.qty !== 'INF' && ticket.qty !== Infinity && parseInt(ticket.qty, 10) > 0;
+const validFiniteQuantity = (ticket: Ticket): boolean => {
+	return ticket.quantity !== Infinity && Math.round(ticket.quantity) > 0;
```

Comment:
nice

also just noticed this method while checking the data type for `Infinity`:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite

Reply:
Adjusted: https://github.com/eventespresso/event-espresso-core/pull/2150/commits/20e1781c6e2165ff384a967bcd31d7883a2eae84

Reply:
Also added some checking for sanity:
![image](https://user-images.githubusercontent.com/1477453/71671724-71ffb700-2d7c-11ea-8407-250a3597ce76.png)